packages/components/src/Toast/Toast.css

```diff
@@ -5,20 +5,25 @@
 
 .container {
   display: flex;
-  flex-direction: column;
-  align-items: center;
   position: fixed;
+  right: 0;
   bottom: var(--space-large);
-  left: 50%;
+  left: 0;
   z-index: var(--elevation-toast);
+  width: 100%;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 var(--space-large);
   color: var(--color-greyBlue--dark);
   font-size: var(--typography--fontSize-large);
-  transform: translateX(-50%);
+  pointer-events: none;
```

Comment:
This is probably the spiciest thing I'm doing in this PR... it simplifies the flex layout and positioning, and lets us kick the transform out though

Reply:
Yeah, does this effectively remove the hit area from the whole toast, and just make it the dismiss button? I think I am still ok with that because we are self-dismissing these. Any accessibility concerns here?

Reply:
This allows for click-through _behind_ the transparent container for `Toast` - then on the Toast itself I'm adding back the pointer-events so the user can select text, dismiss, etc. So the actual hit areas aren't changing, just making sure that if there was some element the user is trying to interact with that's behind the container, the user can still do so!